lib/ansible/parsing/splitter.py

```diff
@@ -87,7 +98,7 @@ def parse_kv(args, check_raw=False):
 
                 # FIXME: make the retrieval of this list of shell/command
                 #        options a function, so the list is centralized
-                if check_raw and k not in ('creates', 'removes', 'chdir', 'executable', 'warn'):
+                if check_raw and k not in _COMMAND_ARGS:
```

Comment:
what we really need at this point is a 'dictionary lookup' of the action since not all of them have the same options

Reply:
@bcoca How about if we just go back to adding only `stdin` in this PR, which is common across all the plugins, and leave a more extensive refactoring for a future PR?  Because ultimately trying to keep this list of keywords in sync with the modules seems like a losing battle, and I wonder if there is a better way...but I'd still like to see `stdin` support sooner rather than later.

Reply:
better way is 'have no hardcoded exceptions', one way to do that is make it 'property based' on the modules, but that would require reading them on controller side.